lib/panda_motd/default_config.yaml

```diff
@@ -16,12 +16,14 @@ components:
   #     can also use 'default' for your default terminal color. Additionally,
   #     you can prefix any of the base colors with 'light_' to get the lighter
   #     version.
+  #   command: The command to generate the text. Defaults to hostname.
```

Comment:
Saying "defaults to hostname" I think might be a bit misleading in this case. If you completely left out the "command" configuration then the script crashes. I think we should remove the text here.

That said, this does bring up the fact that basically none of these values (for any component, for that matter) have true defaults. They'd all break the script if any of them were left. Maybe that's a good thing, maybe not.

Reply:
You think this comment line should be removed altogether?